tests/test_c_generator.py

```diff
@@ -336,8 +336,17 @@ def test_array_decl(self):
         self._assert_ctoc_correct('int g(const int a[const 20]){}')
         ast = parse_to_ast('const int a[const 20];')
         generator = c_generator.CGenerator()
-        self.assertEqual(generator.visit(ast.ext[0].type), 'const int[const 20]')
+        self.assertEqual(generator.visit(ast.ext[0].type), 'const int [const 20]')
         self.assertEqual(generator.visit(ast.ext[0].type.type), 'const int')
 
+    def test_ptr_decl(self):
+        src = 'const int ** const  x;'
+        self._assert_ctoc_correct(src)
+        ast = parse_to_ast(src)
+        generator = c_generator.CGenerator()
+        self.assertEqual(generator.visit(ast.ext[0].type),'const int ** const')
```

Comment:
Please have a space after the comma in argument lists. Wrap the second argument if it doesn't fit in 80 lines (see examples elsewhere)

Reply:
Personally I think that the strict 80 character limitation is archaic and irrelevant with today's display sizes.  
Anyway... Fixed on 878527b.